Makefile

```diff
@@ -47,136 +26,92 @@ all: help
-
-.PHONY: conda-clean
-conda-clean: conda-check
-	@echo "Removing conda env $(CONDA_ENV)"
-	@-"$(CONDA_BIN)" remove -n "$(CONDA_ENV)" --yes --all
-
-.PHONY: conda-spec
-spec: conda-check
-	@echo "Updating conda environment specification file ..."
-	@-"$(CONDA_BIN)" list -n "$(CONDA_ENV)" --explicit > spec-file.txt
-
-## Version targets
-
-.PHONY: bump-dep
-bump-dep: conda-check
-	@-bash -c '$(CONDA_CMD) pip install bump2version'
-
-.PHONY: bump
-bump: conda-check bump-dep
-	@-echo "Updating package version ..."
-	@[ "${VERSION}" ] || ( echo ">> 'VERSION' is not set"; exit 1 )
-	@-bash -c '$(CONDA_CMD) bump2version $(BUMP_XARGS) --new-version "${VERSION}" patch;'
-
 ## Build targets
 
-.PHONY: bootstrap
-bootstrap: conda-check conda-update
-	@echo "Bootstrap ..."
-
-.PHONY: bootstrap-dev
-bootstrap-dev: conda-check conda-update
-	@echo "Bootstrap for development ..."
-
 .PHONY: install
-install: conda-check bootstrap
+install:
 	@echo "Installing application ..."
-	@-bash -c '$(CONDA_CMD) pip install -e "$(APP_ROOT)"'
+	@-bash -c 'pip install -e .'
 	@echo "\nStart service with \`make start'"
 
-.PHONY: install-dev
-install-dev: conda-check bootstrap-dev
+.PHONY: develop
+develop:
 	@echo "Installing development requirements for tests and docs ..."
-	@-bash -c '$(CONDA_CMD) pip install -e "$(APP_ROOT)[dev]"'
+	@-bash -c 'pip install -e .[dev]'
 
-.PHONY: db
-db: conda-check
+.PHONY: migrate
+migrate:
 	@echo "Upgrade or initialize database ..."
-	@-bash -c '$(CONDA_CMD) alembic -c "$(INI_FILE)" upgrade head'
-	@-bash -c '$(CONDA_CMD) initialize_twitcher_db "$(INI_FILE)"'
+	@-bash -c 'alembic -c "$(INI_FILE)" upgrade head'
+	@-bash -c 'initialize_twitcher_db "$(INI_FILE)"'
 
 .PHONY: start
-start: conda-check
+start:
 	@echo "Starting application ..."
-	@-bash -c '$(CONDA_CMD) pserve "$(INI_FILE)" &'
+	@-bash -c 'pserve "$(INI_FILE)"'
 
 .PHONY: clean
-clean: srcclean conda-clean
-	@echo "Cleaning generated files ..."
-	@-for i in $(TEMP_FILES); do \
-  	test -e $$i && rm -v -rf $$i; \
-  done
-
-.PHONY: srcclean
-srcclean:
-	@echo "Removing *.pyc files ..."
-	@-find "$(APP_ROOT)" -type f -name "*.pyc" -print | xargs rm
+clean: clean-build clean-pyc clean-test
+
+.PHONY: clean-build
+clean-build:
+	@echo "Remove build artifacts ..."
+	@-rm -fr build/
+	@-rm -fr dist/
+	@-rm -fr .eggs/
+	@-find . -name '*.log' -exec rm -fr {} +
+	@-find . -name '*.sqlite' -exec rm -fr {} +
+	@-find . -name '*.egg-info' -exec rm -fr {} +
+	@-find . -name '*.egg' -exec rm -f {} +
+
+.PHONY: clean-pyc
+clean-pyc:
+	@echo "Remove Python file artifacts ..."
+	@-find . -name '*.pyc' -exec rm -f {} +
+	@-find . -name '*.pyo' -exec rm -f {} +
+	@-find . -name '*~' -exec rm -f {} +
+	@-find . -name '__pycache__' -exec rm -fr {} +
+
+.PHONY: clean-test
+clean-test:
+	@echo "Remove test and coverage artifacts ..."
+	@-rm -f .coverage
+	@-rm -fr ./coverage/
+	@-rm -fr .pytest_cache
 
 .PHONY: distclean
 distclean: clean
```

Comment:
rename to `clean-dist` for consistency with other targets

Reply:
done.